### PR TITLE
Cleanup cef_dll_wrapper

### LINKED ISSUE #125: [beta] libcef_dll_wrapper.a
Is there a reason to have libcef_dll_wrapper.a in the beta flatpak? This is a static library and it weigth 50MB (which is the size of the current stable flatpak)

### PR DESCRIPTION
**Pull request by hfiguiere**
Closes #125

If it's working ok. (works here, but I could be missing something)

### CODE REVIEW

File: com.obsproject.Studio.json [hunk 1]
```diff
@@ -288,6 +289,7 @@
                 "cp -R ./libcef_dll_wrapper/libcef_dll_wrapper.a /app/cef/libcef_dll_wrapper"
             ],
             "cleanup": [
+                "*.a",
                 "./*"
```
Comment (hfiguiere) [inline com.obsproject.Studio.json:293]: To be fair I'm not sure what's the goal of this cleanup line, I left it alone.

**Comment by hfiguiere** [comment]
This last patch is mandatory. Don't specify branch and commit at the same time.

**Comment by GeorgesStavracas** [comment]
Uh, I think I accidentally force-pushed your branch to beta... do you want me to revert, and you update this PR, so that the authorship is preserved?

**Comment by GeorgesStavracas** [comment]
As agreed on IRC: it's my commit, my fault, and my responsibility now :slightly_smiling_face:

**Pull request closed without merging**